cmd/router/main.go

```diff
@@ -411,10 +415,158 @@ func compilePatterns(patterns []string) ([]*regexp.Regexp, error) {
 }
 
 var (
+	enableAuthFlag         = flag.Bool("enable-auth", false, "protect the inference graph with authorization")
+	graphName              = flag.String("inferencegraph-name", "", "the name of the associated inference graph Kubernetes resource")
 	jsonGraph              = flag.String("graph-json", "", "serialized json graph def")
 	compiledHeaderPatterns []*regexp.Regexp
 )
 
+// findBearerToken parses the standard HTTP Authorization header to find and return
```

Comment:
could we create or add this flow to a doc or README?

Reply:
I can add a mention here: https://github.com/opendatahub-io/kserve/blob/master/docs/odh/authorization.md
In a follow-up PR?

Reply:
sounds good